client/src/components/molecules/AlumniSettings/index.tsx

```diff
@@ -157,8 +158,50 @@ const SettingsModal: FC<Props> = (props): JSX.Element => {
           resolver: yupResolver(SecurityFormSchema)
         })
 
+        // P.S.: This will handle Update the Security Password of the User
+        const handleUpdateSecurity = async (data: Security): Promise<void> => {
+          const { current_password, new_password, password_confirmation } = data
+          const payload = {
+            currentPassword: current_password,
+            newPassword: new_password,
+            newConfirmedPassword: password_confirmation
+          }
+          await axios
+            .put('/api/user/change-password', payload)
```

Comment:
Remove await if you are going to use the `then` and `catch` after the promise is resolved.